src/main/java/nextstep/fp/Lambda.java

```diff
@@ -18,39 +18,10 @@ public static void printAllLambda(List<Integer> numbers) {
     }
 
     public static void runThread() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                System.out.println("Hello from thread");
-            }
-        }).start();
-    }
-
-    public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
-    }
-
-    public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        new Thread(() -> System.out.println("Hello from thread")).start();
     }
 
-    public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        return numbers.stream().filter(c::test).mapToInt(Integer::new).sum();
```

Comment:
```suggestion
        return numbers.stream()
                .filter(c::test)
                .mapToInt(Integer::valueOf)
                .sum();
```
1. 가독성을 위해서 라인마다 한칸씩 내려주시면 좋을 것 같아요 :) 
2. Integer의 생성자는 내부 로직을 들여다보면 사용하지 말라는 의미의 `@Deprecated`가 달려있어요. 내부적으로 자주사용하는 값은 캐싱하여 사용하도록 하고 있기 때문에 `valueOf`를 사용해주시는 것이 좋습니다.

Reply:
좋은 지적 감사드립니다. 사용하는 메서드 내부 로직 확인하는 습관도 들이는 게 좋겠네요!